Dockerfile

```diff
@@ -11,6 +11,8 @@ RUN apt-get update -q && apt-get install -y \
 	libicu-dev \
 	libzip-dev \
 	unzip \
+      tesseract-ocr \
+      libtesseract-dev \
```

Comment:
Do need `libtesseract-dev`?

Reply:
It wasn't working for whatever reason without the lib. I removed and it worked fine. Thanks!

Reply:
Could possibly use `tesseract-ocr-all` instead, or maybe add some select language packages? I think by default it's just osd and eng, but we'll want to test on other languages.